Snippets/Snippets_6/Snippets_6.csproj

```diff
@@ -558,6 +579,8 @@
       <Name>Operations</Name>
     </ProjectReference>
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
```

Comment:
Visual studio adds that for the non-R# test runner. I can remove if wanted, but it's fairly innocuous - and automatic unfortunately.